beamer/config/commands.py

```diff
@@ -30,6 +30,10 @@
 _CONFIG_UPDATE_EVENTS = frozenset((ChainUpdated, FeesUpdated, TokenUpdated, LpAdded, LpRemoved))
 
 
+def _is_config_update_event(event: Event) -> bool:
+    return type(event) in _CONFIG_UPDATE_EVENTS
```

Comment:
nitpick: this could be misleading, I could do updateChain with the exact same values resulting in `_is_config_update_event ` to be true. Or the values could have been changed and then changed back. 
I think it would be more appropriate to check the original state with the state after applying new events and evaluate based on this. Depending on how much effort it is...